be/dtm0_log.h

```diff
@@ -217,26 +213,22 @@ M0_INTERNAL void m0_be_dtm0_log_credit(enum m0_be_dtm0_log_credit_op op,
                                        struct m0_be_seg             *seg,
                                        struct m0_be_tx_credit       *accum);
 // create/destroy
-M0_INTERNAL void m0_be_dtm0_log_create(struct m0_be_tx i      *tx,
+M0_INTERNAL void m0_be_dtm0_log_create(struct m0_be_tx        *tx,
                                        struct m0_be_seg       *seg,
                                        struct m0_be_dtm0_log **out);
 
 M0_INTERNAL void m0_be_dtm0_log_destroy(struct m0_be_dtm0_log **log,
                                         struct m0_be_tx        *tx);
 
 // operational interfaces
-M0_INTERNAL void m0_be_dtm0_log_update(struct m0_be_dtm0_log *log,
-                                       struct m0_be_tx       *tx,
-                                       struct m0_dtm0_txr    *txr);
+M0_INTERNAL void m0_be_dtm0_log_update(struct m0_be_dtm0_log  *log,
+                                       struct m0_be_tx        *tx,
+                                       struct m0_dtm0_clk_src *cs,
```

Comment:
Probably, we could put into the "init" function:

```c
struct m0_be_dtm0_log {
// ...
struct m0_dtm0_clk_src *dl_cs;
};
m0_be_dtm0_log_init(struct m0_be_dtm0_log *log, struct m0_dtm0_clk_src *cs)
{
  log->dl_cs = cs;
}
```

Reply:
Added at f3411c64e7a2014ecd0516ed3f5108b802f03e57